gempak/ush/gdas_ecmwf_meta_ver.sh

```diff
@@ -195,28 +136,28 @@ EOFplt
     done
 done
 
-export err=$?;err_chk
+export err=$?
+
 #####################################################
 # GEMPAK DOES NOT ALWAYS HAVE A NON ZERO RETURN CODE
 # WHEN IT CAN NOT PRODUCE THE DESIRED GRID.  CHECK
 # FOR THIS CASE HERE.
 #####################################################
-ls -l ecmwfver.meta
-export err=$?;export pgm="GEMPAK CHECK FILE";err_chk
-
-if [ $SENDCOM = "YES" ] ; then
-    mkdir -p -m 775 ${COMOUTecmwf}.${PDY}/meta
-    mv ecmwfver.meta ${COMOUTecmwf}.${PDY}/meta/ecmwfver_${PDY}_${cyc}
-    export err=$?
-    if [[ $err -ne 0 ]] ; then
-       echo " File ecmwfver.meta does not exist."
-       exit $err
-    fi
+if (( err != 0 )) || [[ ! -s ecmwfver.meta ]]; then
+    echo "FATAL ERROR: Failed to create ecmwf meta file"
+    exit "${err}"
+fi
 
-    if [ $SENDDBN = "YES" ] ; then
-        ${DBNROOT}/bin/dbn_alert MODEL ECMWFVER_HPCMETAFILE $job \
-        ${COMOUTecmwf}.${PDY}/meta/ecmwfver_${PDY}_${cyc}
-    fi
+mv ecmwfver.meta "${COM_ATMOS_GEMPAK_META}/ecmwfver_${PDY}_${cyc2}"
```

Comment:
Not sure why this script creates a ecmwf meta file. If ${COM_ATMOS_GEMPAK_META} is pointed to gdas.20240310/00/atmos/gempak/meta, there is no ecmwfver_20240310_00 file under /lfs/h1/ops/prod/com/gfs/v16.3/gdas.20240310/00/atmos/gempak/meta. In the old code, it is sent to ${COMOUTecmwf}.${PDY}/meta. I don't see it under /lfs/h1/ops/prod/com/ecmwf/v2.1/ecmwf.20240310/gempak either.

Reply:
This file is generated for the 06 cycle only. It is located at /lfs/h1/ops/prod/com/gfs/v16.3/gdas.20240315/06/atmos/gempak/meta.20240315/meta.